packages/strapi-plugin-content-type-builder/admin/src/components/TableListRow/index.js

```diff
@@ -38,19 +37,16 @@ class TableListRow extends React.Component { // eslint-disable-line react/prefer
     router.push(`/plugins/content-type-builder/models/${this.props.rowItem.name}`);
   }
 
-  toggleModalWarning = () => {
-    this.setState({ showWarning: !this.state.showWarning });
-  }
+  toggleModalWarning = () => this.setState({ showWarning: !this.state.showWarning });
 
-  handleShowModalWarning = (e) => {
-    e.stopPropagation();
-    this.setState({ showWarning: !this.state.showWarning });
-  }
+  handleShowModalWarning = () => this.setState({ showWarning: !this.state.showWarning });
 
   render() {
     const temporary = this.props.rowItem.isTemporary ? <FormattedMessage id="content-type-builder.contentType.temporaryDisplay" /> : '';
     const description = isEmpty(this.props.rowItem.description) ? '-' :  this.props.rowItem.description;
     const spanStyle = this.props.rowItem.isTemporary ? '60%' : '100%';
+    const icons = [{ icoType: 'pencil', onClick: this.handleEdit }, { onClick: this.handleShowModalWarning }];
```

Comment:
I would prefer to say explicitly the second icon is a `trash`. There are many reasons:

- What happens if we decided to change the default icon in the future? You will have to come back here and update this line of code.
- It allows any developer to understand without reading precisely the documentation.